lib/Dancer2/CLI/Gen.pm

```diff
@@ -149,13 +163,35 @@ following commands:
 }
 
 sub _how_to_run {
-    my( $self, $app_path ) = @_;
-    print qq{
+    my( $self, $vars ) = @_;
+
+    my $app_path = $vars->{ apppath };
+    my $app_name = $vars->{ appname };
+    if( $vars->{ docker } ) {
+        my $image = lc $app_name;
+        print qq{
 Your new application is ready! To run it:
 
+        cd $app_path
+        docker build -t ${image} .
+        docker run -d -p 5000:5000 --name $app_name ${image}
+
+(note: you may need to run the docker commands with sudo)
+
+You may also run your app without Docker:
+
         cd $app_path
         plackup bin/app.psgi
+};
+    } else {
+        print qq{
+Your new application is ready! To run it:
 
+        cd $app_path
+        plackup bin/app.psgi
```

Comment:
It feels like this bit could be in a var like `$plackup_cmds` or similar that both branches of the if can use, rather than copied & pasted - just so that if that bit wants changing in future we don't have to remember to change both.  Then again, it's right next to each other, so not that hard!

Reply:
I eliminated the redundancy. Thanks for spotting that!